graphql/schema.graphql

```diff
@@ -58,17 +69,105 @@ type Gun {
     character: Character @belongsTo
     created_at: DateTime!
     updated_at: DateTime!
+    json_stats: String!
```

Comment:
No op for now, but I think we can make a custom type for json if there isn't one provided in the lighthouse package.

Reply:
There may be one, I was just defaulting to string here as that's how it's saved in the DB. Similar to your previous point about JSON, whichever approach we use we just have to be consistent.

Reply:
Reverted the datatype to STRING in the Database, but using custom JSON Scalar in the GraphQL